Tests/Utilities.UI/UI/InteractiveWindow.cs

```
@@ -281,25 +298,44 @@ public InteractiveWindow(string title, AutomationElement element, VisualStudioAp
             }
         }
 
-        public void Reset() {
-            Console.WriteLine("REPL resetting");
-
-            Assert.IsTrue(ReplWindow.Reset().Wait(10000));
+        public string PrimaryPrompt {
+            get {
+#if DEV14_OR_LATER
+#if NTVS_FEATURE_INTERACTIVEWINDOW
+#error Implement for NTVS
+#else
+                return (ReplWindow.Evaluator as BasePythonReplEvaluator)?.PrimaryPrompt ?? ">>>";
+#endif
+#else
+                return (string)ReplWindow.GetOptionValue(ReplOptions.CurrentPrimaryPrompt);
+#endif
+            }
         }
 
-        public void WithStandardInputPrompt(string prompt, Action<string> action) {
-            if ((bool)ReplWindow.GetOptionValue(ReplOptions.DisplayPromptInMargin)) {
-                action(String.Empty);
-                return;
+        public string SecondaryPrompt {
+            get {
+#if DEV14_OR_LATER
+#if NTVS_FEATURE_INTERACTIVEWINDOW
+#error Implement for NTVS

```

do we really need to throw an error here? Can we use ```#if DEV14_OR_LATER && !NTVS_FEATURE_INTERACTIVEWINDOW``` instead?